concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetAccountNonFinalizedTransactionsTest.java

```diff
@@ -0,0 +1,70 @@
+package com.concordium.sdk;
```

Comment:
What is this example testing compared to the `concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetAccountNonFinalizedTransactions.java`?

Reply:
* The tests mock the grpc service and therefore can be executed without a node. 
* Tests allows to provide required inputs and tests them on various constraints. For example the account address used in the examples may / may not have any non finalized transactions when the example is executed.